src/main/java/Directory.java

```diff
@@ -0,0 +1,40 @@
+import java.io.BufferedReader; 
+import java.io.File;
+import java.io.FileReader; 
+import java.io.IOException; 
+
+public class Directory implements DataStore {
+  private File directory;
+  private String directoryPath;
+
+  public Directory(String directoryPath) throws NonexistentDirectoryException {
+    File directory = new File(directoryPath);
+    if (directory.exists()) {
+      this.directoryPath = directoryPath; 
+    } else {
+      throw new NonexistentDirectoryException(directoryPath);
+    }
+  }
+
+  public Boolean existsInStore(String uri) {
+    String filePath = this.directoryPath + uri;
+    return new File(filePath).exists() ? true : false;
```

Comment:
You can just return `new File(filePath).exists()` here since it returns a boolean.